apps/meteor/tests/e2e/omnichannel/omnichannel-livechat-fileupload.spec.ts

```diff
@@ -0,0 +1,123 @@
+	[{ url: '/settings/Livechat_fileupload_enabled', value: false}],
+	[{ url: '/settings/FileUpload_Enabled', value: false}, { url: '/settings/Livechat_fileupload_enabled', value: false}],
+]
+
+const beforeTest = async (poLiveChat: OmnichannelLiveChat) => {
+	await poLiveChat.page.goto('/livechat');
+
+	await poLiveChat.openAnyLiveChat();
+	await poLiveChat.sendMessage(visitor, false);
+	await poLiveChat.onlineAgentMessage.fill('this_a_test_message_from_user');
+	await poLiveChat.btnSendMessageToOnlineAgent.click();
+
+	await poLiveChat.txtChatMessage('this_a_test_message_from_user').waitFor({state: 'visible'});
+}
+
+test.describe('OC - Livechat - OC - File Upload', () => {
+	let poLiveChat: OmnichannelLiveChat;
+	let poHomeOmnichannel: HomeOmnichannel;
+	let agent: Awaited<ReturnType<typeof createAgent>>;
+
+	test.beforeAll(async ({ browser, api }) => {
+		agent = await createAgent(api, 'user1');
+
+		const { page } = await createAuxContext(browser, Users.user1, '/');
+		poHomeOmnichannel = new HomeOmnichannel(page);
+	});
+
+	test.beforeEach(async ({ page, api }) => {
+		poLiveChat = new OmnichannelLiveChat(page, api);
+	});
+
+	test.afterAll(async ({api}) => {
+		await api.post('/settings/FileUpload_Enabled', { value: true });
+		await api.post('/settings/Livechat_fileupload_enabled', { value: true });
+
+		await poHomeOmnichannel.page?.close();
+		await agent.delete();
+	});
+
+	// Default settings are FileUpload_Enabled true and Livechat_fileupload_enabled true
+	test('OC - Livechat - txt Drag & Drop', async () => {
+		await beforeTest(poLiveChat);
+
+		await test.step('expect to upload a txt file', async () => {
+			await poLiveChat.dragAndDropTxtFile();
+			await expect(poLiveChat.findUploadedFileLink('any_file.txt')).toBeVisible();
+		});
+	});
+
+	test('OC - Livechat - lst Drag & Drop', async () => {
+		await beforeTest(poLiveChat);
+
+		await test.step('expect to upload a txt file', async () => {
+			await poLiveChat.dragAndDropLstFile();
+			await expect(poLiveChat.findUploadedFileLink('lst-test.lst')).toBeVisible();
+		});
+	});
+});
+
+test.describe('OC - Livechat - OC - File Upload - Disabled', () => {
+	let poLiveChat: OmnichannelLiveChat;
+	let poHomeOmnichannel: HomeOmnichannel;
+	let agent: Awaited<ReturnType<typeof createAgent>>;
+
+	test.beforeAll(async ({ browser, api }) => {
+		agent = await createAgent(api, 'user1');
+
+		const { page } = await createAuxContext(browser, Users.user1, '/');
+		poHomeOmnichannel = new HomeOmnichannel(page);
+	});
+
+	test.afterAll(async ({api}) => {
+		await api.post('/settings/FileUpload_Enabled', { value: true });
+		await api.post('/settings/Livechat_fileupload_enabled', { value: true });
+
+		await poHomeOmnichannel.page?.close();
+		await agent.delete();
+	});
+
+	endpointMatrix.forEach((endpoints) => {
+		const testName = endpoints.map((endpoint) => endpoint.url.split('/').pop()).join(' ');
+
+		test(`OC - Livechat - txt Drag & Drop - ${testName}`, async ({ page, api }) => {
```

Comment:
We could add the value here as well, since now it will be something like: `OC - Livechat - txt Drag & Drop - FileUpload_Enabled Livechat_fileupload_enabled`.

 It would be clearer if `OC - Livechat - txt Drag & Drop - FileUpload_Enabled=false Livechat_fileupload_enabled=true`